### PR TITLE
Add standard_form modifications

### LINKED ISSUE #1890: Modify RHS (and define RHS)
In #1183 we decided to not implement: `ScalarConstantChange`.
Probably following: #470 
According to the manual (http://www.juliaopt.org/JuMP.jl/v0.19.0/constraints/#Modifying-a-constant-term-1):
In this constraint: `@constraint(model, 2x + 1 <= x - 3)`, why the coefficient of `x` is well defined and the constant of the function is not?

Moreover, in #1332 we are considering bounds on RHS, but we have not defined RHS.

Maybe we should reconsider?
At least bring back `add_constant` (or `add_to_rhs`)? https://github.com/JuliaOpt/JuMP.jl/pull/1351#discussion_r224945107

### PR DESCRIPTION
**Pull request by odow**
Closes #1890

I have deprecated `set_coefficient` in favor of `set_standard_form_coefficient`.

### CODE REVIEW

File: src/constraints.jl [hunk 1]
```diff
@@ -395,36 +395,94 @@ function add_constraint(model::Model, c::AbstractConstraint, name::String="")
...
+    MOI.modify(backend(owner_model(constraint)), index(constraint),
+               MOI.ScalarCoefficientChange(index(variable), convert(T, value)))
+    return
+end
+@deprecate set_coefficient set_standard_form_coefficient
+
+"""
+    standard_form_coefficient(constraint::ConstraintRef, variable::VariableRef)
+
+Return the coefficient associated with `variable` in `cosntraint` after JuMP has
```
Comment (joaquimg) [inline src/constraints.jl]: constraint

**Comment by blegat** [comment]
I don't support renaming `set_coefficient` to set_standard_form_coefficient`.
The "standard form" is confusing because of the confusion with https://en.m.wikipedia.org/wiki/Linear_programming#Standard_form

**Comment by odow** [comment]
Not sure if this counts as breaking because it adds deprecations.

**Comment by blegat** [comment]
Do you think it should be included in v0.19.1 ?

**Comment by odow** [comment]
I don't mind. It isn't urgent, so it can probably wait. (Selfish reason, I'll have to update SDDP.jl, and relying on a patch 0.19.1 isn't fun.)

**Comment by blegat** [comment]
Even if there is warning, if someone release a package using this function and people use this package, when updating JuMP, the package will throw warning so it is kind of breaking.
I'd like to release MOI v0.9 in a few days so if we give a week for packages to update to it, we can release JuMP v0.20 in about a week.

**Pull request merged**